src/url_search_params.rs

```diff
@@ -137,10 +138,7 @@ impl URLSearchParams {
     pub fn keys(&self) -> Array {
         self.params
             .iter()
-            .map(|p| &p.0)
-            .collect::<HashSet<_>>()
-            .into_iter()
-            .map(JsValue::from)
+            .map(|p| JsValue::from(&p.0))
```

Comment:
This does not return unique values. Since `append` can create multiple same keys, we need to have some or data structure/method to remove recurring keys in the list

Reply:
I checked/tested native implementations of other runtimes (Node.js and browsers) and `keys()` does not filter duplicated keys. So I think it is expected to have as output duplucates. Am I wrong?

Reply:
My bad! I've misread the documentation.

```
Welcome to Node.js v17.5.0.
Type ".help" for more information.
> const params = new URLSearchParams('hello=world&hello=second')
undefined
> params.keys()
URLSearchParams Iterator { 'hello', 'hello' }
```